model/internal/data/pdatacontext.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package data
+
+type PDataContext struct {
+	List []PDataContextKeyValue
```

Comment:
Can we use the normal "AnyValue"/Attributes?

Reply:
Not for the Key, it needs to be a comparable interface{}, like it is done for `context.Context`, but for the Value it is an interesting idea, since it may make some code that needs to deal with both telemetry attributes and PDataContext easier to write. It is worth exploring.

Reply:
Did you mean pdata.AttributeValue? If so, that would cause an import cycle that is not allowed.